### PR TITLE
MF-142: Update Vitals Navigation and Structure

### PR DESCRIPTION
**Pull request by denniskigen**
This PR modifies the vitals widget so that it corresponds with the routing and navigation structure specified in https://issues.openmrs.org/projects/MF/issues/MF-142.

Screenshot 2020-03-31 at 12 00 36
Screenshot 2020-03-31 at 12 01 17
Screenshot 2020-03-31 at 12 01 25
Screenshot 2020-03-31 at 12 01 32

### CODE REVIEW

File: src/widgets/vitals/vitals.test.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { cleanup, render, wait } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import Vitals from "./vitals.component";
+
+describe("<NotesComponent />", () => {
```
Comment (donaldkibet) [inline src/widgets/vitals/vitals.test.tsx]:
```suggestion
describe("<Vitals />", () => {
```

File: src/widgets/vitals/vitals-detailed-summary.css [hunk 2]
```diff
@@ -33,18 +44,20 @@
...
+
 .vitalsFooter svg {
   margin-right: 0.5rem;
 }
 
 :global(.omrs-breakpoint-gt-phone) .vitalsSummary {
   display: flex;
   justify-content: center;
-  padding-top: 3.25rem;
+  /* padding-top: 3.25rem;  */
```
Comment (donaldkibet) [inline src/widgets/vitals/vitals-detailed-summary.css]:
```suggestion

```

**Review by donaldkibet — APPROVED** [review]
Nice work, just a few suggestions

File: src/widgets/vitals/vitals.test.tsx (hunk 1 above)
Comment (denniskigen) [reply src/widgets/vitals/vitals.test.tsx]: Fixed

File: src/widgets/vitals/vitals-detailed-summary.css (hunk 2 above)
Comment (denniskigen) [reply src/widgets/vitals/vitals-detailed-summary.css]: Fixed

**Pull request merged**